packages/angular_devkit/build_angular/package.json

```diff
@@ -67,7 +67,7 @@
     "@angular/platform-server": "^8.0.0-beta.9",
     "@angular/router": "^8.0.0-beta.9",
     "@angular/service-worker": "^8.0.0-beta.9",
-    "codelyzer": "^4.2.1",
+    "codelyzer": "^5.0.0",
```

Comment:
@clydin I didn't find the corresponding tslint file here. Let me know if there are other migrations necessary.

Reply:
That is there for dev/testing purposes only.  No need for further action.